Incomplete winget command for MSVC install on Windows 10

### Environment

- OS and Version: Windows_NT x64 10.0.19045
- VS Code Version: 1.97.2
- C/C++ Extension Version: 1.23.6

### Bug Summary and Steps to Reproduce

Bug Summary:

The winget command given on the walkthrough `winget install Microsoft.VisualStudio.2022.BuildTools --force --override "--wait --passive --add Microsoft.VisualStudio.Workload.VCTools --add Microsoft.VisualStudio.Component.VC.Tools.x86.x64 --add Microsoft.VisualStudio.Component.Windows10SDK"` installs an incomplete MSVC compiler that is unable to run the hello world example given on the walkthrough.

Steps to reproduce:
1. Go to C++ walkthrough "Get Started with C++ Development"
2. Execute the winget command given to install the Microsoft Visual C++ (MSVC) compiler
3. Create the C++ file just like shown in the picture of step 2 of the walkthrough (the code is in the additional context section of this issue)
4. See error about missing float.h from the `#include <iostream>`

Expected behavior:

Correct and complete installation of the MSCV compiler, which can execute the provided hello word example.

Temporary fix:

1. Reinitialize the `Visual Studio Installer`
2. Modify the installation of Visual Studio Build Tools 2022
3. Deselect and select again the `Desktop development with C++`. There should be an additional 3GB of files to be installed 
4. Install

Basically, follow the instructions present in https://code.visualstudio.com/docs/cpp/config-msvc

### Configuration and Logs

```shell
Not applicable
```

### Other Extensions

_No response_

### Additional context

Code in question of `hello-world.cpp`:

```cpp
#include <iostream>

int main() {
    std::cout << "Hello, World!" << std::endl;
    return 0;
}
```

Returns

```pwsh
C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.43.34808\include\cfloat(10): fatal error C1083: Cannot open include file: 'float.h': No such file or directory
```

When compiled with MSVC compiler.

PS: On step 3 of the walkthrough "Launch from the Developer Command Prompt for VS" displays a blue button that restarts the VSC window, yet the instructions require that you close VSC and start it from development cmd or pwsh.

--- a/Extension/walkthrough/installcompiler/install-compiler-windows10.md
+++ b/Extension/walkthrough/installcompiler/install-compiler-windows10.md
@@ -2,7 +2,7 @@
 <p data-loc-id="walkthrough.windows.text1">If you&#39;re doing C++ development for Windows, we recommend installing the Microsoft Visual C++ (MSVC) compiler.</p>
 <ol>
 <li><p data-loc-id="walkthrough.windows.text2">To install MSVC, open the VS Code terminal (CTRL + `) and paste in the following command:
-<pre><code style="white-space: pre-wrap;">winget install Microsoft.VisualStudio.2022.BuildTools --force --override "--wait --passive --add Microsoft.VisualStudio.Workload.VCTools --add Microsoft.VisualStudio.Component.VC.Tools.x86.x64 --add Microsoft.VisualStudio.Component.Windows10SDK"</code></pre>
+<pre><code style="white-space: pre-wrap;">winget install Microsoft.VisualStudio.2022.BuildTools --force --override "--wait --passive --add Microsoft.VisualStudio.Workload.VCTools --add Microsoft.VisualStudio.Component.VC.Tools.x86.x64 --add Microsoft.VisualStudio.Component.Windows10SDK.20348"</code></pre>
 </li>
 <blockquote>
 <p><strong data-loc-id="walkthrough.windows.note1">Note</strong>: <span data-loc-id="walkthrough.windows.note1.text">You can use the C++ toolset from Visual Studio Build Tools along with Visual Studio Code to compile, build, and verify any C++ codebase as long as you also have a valid Visual Studio license (either Community, Pro, or Enterprise) that you are actively using to develop that C++ codebase.</span></p>

--- a/Extension/walkthrough/installcompiler/install-compiler-windows11.md
+++ b/Extension/walkthrough/installcompiler/install-compiler-windows11.md
@@ -2,7 +2,7 @@
 <p data-loc-id="walkthrough.windows.text1">If you&#39;re doing C++ development for Windows, we recommend installing the Microsoft Visual C++ (MSVC) compiler.</p>
 <ol>
 <li><p data-loc-id="walkthrough.windows.text2">To install MSVC, open the VS Code terminal (CTRL + `) and paste in the following command:
-<pre><code style="white-space: pre-wrap;">winget install Microsoft.VisualStudio.2022.BuildTools --force --override "--wait --passive --add Microsoft.VisualStudio.Workload.VCTools --add Microsoft.VisualStudio.Component.VC.Tools.x86.x64 --add Microsoft.VisualStudio.Component.Windows11SDK.22000"</code></pre>
+<pre><code style="white-space: pre-wrap;">winget install Microsoft.VisualStudio.2022.BuildTools --force --override "--wait --passive --add Microsoft.VisualStudio.Workload.VCTools --add Microsoft.VisualStudio.Component.VC.Tools.x86.x64 --add Microsoft.VisualStudio.Component.Windows11SDK.26100"</code></pre>
 </li>
 <blockquote>
 <p><strong data-loc-id="walkthrough.windows.note1">Note</strong>: <span data-loc-id="walkthrough.windows.note1.text">You can use the C++ toolset from Visual Studio Build Tools along with Visual Studio Code to compile, build, and verify any C++ codebase as long as you also have a valid Visual Studio license (either Community, Pro, or Enterprise) that you are actively using to develop that C++ codebase.</span></p>

